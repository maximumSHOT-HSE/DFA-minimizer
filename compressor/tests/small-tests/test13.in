16 112 7
1 2 0
1 7 1
1 8 2
1 5 3
1 8 4
1 2 5
1 1 6
2 2 0
2 6 1
2 8 2
2 8 3
2 3 4
2 8 5
2 1 6
3 8 0
3 9 1
3 1 2
3 8 3
3 2 4
3 2 5
3 8 6
4 1 0
4 5 1
4 9 2
4 5 3
4 5 4
4 9 5
4 8 6
5 2 0
5 3 1
5 6 2
5 3 3
5 6 4
5 8 5
5 2 6
6 3 0
6 7 1
6 3 2
6 5 3
6 2 4
6 4 5
6 7 6
7 8 0
7 5 1
7 9 2
7 9 3
7 7 4
7 9 5
7 3 6
8 2 0
8 7 1
8 8 2
8 1 3
8 2 4
8 9 5
8 5 6
9 2 0
9 5 1
9 5 2
9 9 3
9 8 4
9 5 5
9 5 6
10 2 0
10 7 1
10 8 2
10 1 3
10 2 4
10 9 5
10 5 6
11 2 0
11 7 1
11 8 2
11 5 3
11 8 4
11 2 5
11 1 6
12 2 0
12 7 1
12 8 2
12 1 3
12 2 4
12 9 5
12 5 6
13 2 0
13 7 1
13 8 2
13 5 3
13 8 4
13 2 5
13 1 6
14 2 0
14 7 1
14 8 2
14 1 3
14 2 4
14 9 5
14 5 6
15 1 0
15 5 1
15 9 2
15 5 3
15 5 4
15 9 5
15 8 6
16 1 0
16 5 1
16 9 2
16 5 3
16 5 4
16 9 5
16 8 6
9
6
2 9 1 6 7 6 
