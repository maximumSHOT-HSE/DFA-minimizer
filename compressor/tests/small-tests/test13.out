10 70 7
4 1 0
3 1 2
8 1 3
1 1 6
2 1 6
10 1 6
1 2 0
2 2 0
5 2 0
8 2 0
9 2 0
10 2 0
3 2 4
6 2 4
8 2 4
1 2 5
3 2 5
10 2 5
5 2 6
6 3 0
5 3 1
6 3 2
5 3 3
2 3 4
7 3 6
6 4 5
4 5 1
7 5 1
9 5 1
9 5 2
1 5 3
4 5 3
6 5 3
10 5 3
4 5 4
9 5 5
8 5 6
9 5 6
2 6 1
5 6 2
5 6 4
1 7 1
6 7 1
8 7 1
10 7 1
7 7 4
6 7 6
3 8 0
7 8 0
1 8 2
2 8 2
8 8 2
10 8 2
2 8 3
3 8 3
1 8 4
9 8 4
10 8 4
2 8 5
5 8 5
3 8 6
4 8 6
3 9 1
4 9 2
7 9 2
7 9 3
9 9 3
4 9 5
7 9 5
8 9 5
9
5
1 2 6 7 9 
